db/version_set.cc

```diff
@@ -578,6 +586,21 @@ void LevelIterator::Seek(const Slice& target) {
 
   InitFileIterator(new_file_index);
   if (file_iter_.iter() != nullptr) {
+    // TODO(peter): Rather the seeking to target, we could use
```

Comment:
This TODO is done inside the `Seek` call, right?

Reply:
This is detritus now that `BlockBasedTableIterator` skips keys covered by range tombstones. Removed.